src/components/ResourceHeader/index.jsx

```diff
@@ -53,11 +53,11 @@ const ResourceHeader = ({
       <div className="ds-l-row ds-u-align-items--center">
         <div className="ds-l-col--12 ds-u-display--flex ds-u-justify-content--between ds-u-align-items--center">
           <div className="ds-u-font-weight--bold">
-            {/* <DataTablePageResults
+            {<DataTablePageResults
```

Comment:
This should be considered blocked and left commented out. I am working on it in another branch.